src/NodaTime.Test/InstantTest.cs

```diff
@@ -16,20 +16,39 @@ public partial class InstantTest
         private static readonly Instant threeMillion = Instant.FromUntrustedDuration(Duration.FromNanoseconds(3000000L));
         private static readonly Instant negativeFiftyMillion = Instant.FromUntrustedDuration(Duration.FromNanoseconds(-50000000L));
 
+        // When dealing with floating point binary data, if we're accurate to 50 milliseconds, that's fine...
+        // (0.000001 days = ~86ms, as a guide to the scale involved...)
+
         [Test]
-        public void FromJDN()
+        public void FromJulianDate_OriginalTestData()
```

Comment:
I don't know where the original test data came from, unfortunately. @GoodOlClint, can you cast any light on this? I agree that it would be nice to rename the test appropriately. (Would be even nicer to parameterize, but it wouldn't be terribly clean in terms of parameters.)